### PR TITLE
build: set up codeql analysis

### PR DESCRIPTION
**Pull request by jimmyandrade**
This PR proposes adding CodeQL Analysis

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        # Override automatic language detection by changing the below list
+        # Supported options are ['csharp', 'cpp', 'go', 'java', 'javascript', 'python']
+        language: ['javascript']
```
Comment (shilman) [inline .github/workflows/codeql-analysis.yml:27]: Typescript?

**Review by shilman — COMMENTED** [review]
LGTM 👍 cc @gaetanmaisse

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (JackHowa) [reply .github/workflows/codeql-analysis.yml:27]: are you suggesting adding typescript?

Comment (gaetanmaisse) [reply .github/workflows/codeql-analysis.yml:27]: Good try @shilman but this repo only contains JS, plus:
> Supported options are ['csharp', 'cpp', 'go', 'java', 'javascript', 'python']

I knew it! Deep down you're a TS fanboy. 😛 😂

**Review by gaetanmaisse — APPROVED** [review]

**Review by JackHowa — APPROVED** [review]

**Comment by hipstersmoothie** [comment]
:rocket: PR was released in [`v2.8.8`](https://github.com/storybookjs/storybook-deployer/releases/tag/v2.8.8) :rocket:

**Pull request merged**